framework/src/wifi_manager/Kconfig

```diff
@@ -70,5 +70,22 @@ config WIFIMGR_DISABLE_AUTO_GET_IP
 	bool "disable auto get ip (ipv4 dhcp)"
 	default n
 
+config WIFIMGR_DISABLE_EXTERNAL_AUTOCONNECT
```

Comment:
TizenRT supports various wi-fi board. if we add this configuration then other boards also provide this feature. So I think removing dependency of the configuration would be better.

Reply:
@pillip8282 Recently, I have found that autoconnect of the Wi-Fi driver needs be avoided in certain applications. For example, if the autoconnect takes a long time, reconnecting only at the driver-level will prevent the application from making the necessary reconnection (extreme case, DHCP). On the other hand, the autoconnect from the driver sometimes makes the reconnection very easy and straightforward. Therefore, my recommandation is to guide the vender to provide this configuration. Please let me know your opinion. Thanks!